### PR TITLE
Fixing CoreFxLab to build

### PR DESCRIPTION
**Pull request by sokket**
CoreFXLab is totally broken and nothing builds. Fixing things so everything builds again 

@KrzysztofCwalina @krwq

### CODE REVIEW

**Comment by sokket** [comment]
Note - not everything is done yet, very few things don't compile but I figured I'd put this out now since it's a big change

**Comment by KrzysztofCwalina** [comment]
looks good.

File: src/System.Drawing.Graphics/tests/System.Drawing.Graphics.Tests.xproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>C0219E93-1738-45A1-92CA-16693FAAEC1B</ProjectGuid>
+    <RootNamespace>System.Buffers.Tests</RootNamespace>
```
Comment (livarcocc) [inline src/System.Drawing.Graphics/tests/System.Drawing.Graphics.Tests.xproj]: Fix the RootNamespace here.

File: src/System.Reflection.Metadata.Cil/src/project.json [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
+    "System.Runtime": "4.0.21-*",
+    "System.Reflection": "4.0.0-*",
+    "System.Reflection.Primitives": "4.0.1-*",
+    "System.Reflection.Metadata": "1.1.1-*",
+    "System.Reflection.TypeExtensions": "4.0.0-*"
   },
   "frameworks": {
-    "dnxcore50": {
-    }
+    "dnxcore50": { }
```
Comment (livarcocc) [inline src/System.Reflection.Metadata.Cil/src/project.json]: Why aren't we moving this to dotnet5.6 along with the others?

**Comment by livarcocc** [comment]
@sokket Is there a particular reason we check-in the lock files? Could we remove them from the repo at this point?

Minor comments, overall, looks good.

File: src/System.Drawing.Graphics/tests/System.Drawing.Graphics.Tests.xproj (hunk 1 above)
Comment (sokket) [inline src/System.Drawing.Graphics/tests/System.Drawing.Graphics.Tests.xproj]: Fixed

File: src/System.Reflection.Metadata.Cil/src/project.json (hunk 2 above)
Comment (sokket) [inline src/System.Reflection.Metadata.Cil/src/project.json]: I don't have this project compiling yet :( still working on it

**Comment by sokket** [comment]
@livarcocc I'll add the lock files to the gitignore and delete them

**Comment by krwq** [comment]
LGTM :+1:  we can follow up on this once everything is building, considering current state of the repo it significantly improves it - thanks! :)

**Comment by KrzysztofCwalina** [comment]
@dotnet-bot test this please

**Comment by sokket** [comment]
@dotnet-bot test this please

**Pull request merged**